### PR TITLE
added content about Zero Knowledge proofs

### PR DESCRIPTION
**Pull request by FidelVe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,3 +227,7 @@ _Tutorials for participating in the ICON ecosystem_
 ## Templates
 _Templates for projects or other useful things_
 - [Template repository](https://github.com/icon-community/template-repository) - Template repository for a new project, including README, Issue Templates, PR Template, LICENSE, Release message auto-instructions, CONTRIBUTING, and CODE_OF_CONDUCT files
+
+## Zero-knowledge proofs
```
Comment (han-so1omon) [inline README.md]: Needs to be a subcategory of the 'projects' section

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ _Note that smart contracts decentralized applications also function as libraries
   - [Funding programs](#funding-programs)
   - [Content](#content)
   - [Tutorials](#tutorials)
+  - [Zero-knowledge proofs](#zero-knowledge-proofs)
```
Comment (han-so1omon) [inline README.md]: Should be a subcategory of the 'projects' section

**Review by han-so1omon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (FidelVe) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (FidelVe) [reply README.md]: done

**Review by han-so1omon — APPROVED** [review]

**Pull request merged**